.travis.yml

```diff
@@ -1,13 +1,14 @@
 language: scala
-script:
-  - ./bin/runci.sh $CI_TEST
-jdk:
-  - oraclejdk8
+scala: 2.12.3
+jdk: oraclejdk8
+
+script: ./bin/runci.sh "$CI_TEST"
 env:
   global:
     - TRAVIS_NODE_VERSION="8"
+    - TRAVIS_YARN_VERSION="1.1.0"
   matrix:
-    - CI_TEST: test
+    - CI_TEST: ;compile;test
```

Comment:
what is the motivation for this change?

Reply:
Without this yarn fails randomly because it is run in parallel for both compile and test. At least that’s what I think is happening.